### PR TITLE
Change to fix null resource exceptions

### PR DESCRIPTION
**Pull request by jleopold28**
Issue is described here https://github.com/duo-labs/cloudmapper/issues/540

We are getting this same error when running on legacy dev accounts.

### CODE REVIEW

**Review by jantman — APPROVED** [review]
Approved pending you've verified that that command fixes it. I'd really prefer using an upstream commit if they have one merged that fixes the bug, or using a `patch` if we need to change the code like this... but I guess in this one place this should be OK...

**Comment by jleopold28** [comment]
> Approved pending you've verified that that command fixes it. I'd really prefer using an upstream commit if they have one merged that fixes the bug, or using a `patch` if we need to change the code like this... but I guess in this one place this should be OK...

@jantman So I have not verified that the command fixes this. Mainly because I cannot manually run cloudmapper on the legacy dev accounts during the day (due to API limits). My plan was to check if this fixes the issue and then go back and submit a PR on the cloudmapper repo. 

Also there are no upstream commits that fix this issue currently.

**Review by kennyg — APPROVED** [review]

**Comment by jleopold28** [comment]
Closing this for now. Testing changes before merge.

**Pull request merged**
